api/libproject/cloudsqlproxycontext.rb

```diff
@@ -5,23 +5,25 @@
 class CloudSqlProxyContext < ServiceAccountContext
 
   def run()
-    # TODO(dmohs): An error here does not cause the main thread to die.
-    super do
-      common = Common.new
-      @ps = fork do
-        exec *%W{
-          cloud_sql_proxy
-            -instances #{@project}:us-central1:workbenchmaindb=tcp:0.0.0.0:3307
-            -credential_file=#{ServiceAccountContext::SERVICE_ACCOUNT_KEY_PATH}
-        }
-      end
-      begin
-        sleep 1 # TODO(dmohs): Detect running better.
-        yield
-      ensure
-        Process.kill "HUP", @ps
-        Process.wait
-      end
+    @ps = fork do
+      exec *%W{
+        cloud_sql_proxy
+          -instances #{@project}:us-central1:workbenchmaindb=tcp:0.0.0.0:3307
```

Comment:
Please excuse my drive-by comment. I know `us-central1` is the current region, but will this always be true? Should we pull that into a config somewhere?

Reply:
Good question. My philosophy is that we should put it into a config as soon as we have two and no sooner.